packages/heptagon/heptagon.1.00.06/opam

```diff
@@ -29,6 +29,6 @@ following features:
 See http://bzr.inria.fr for further informations."""
 url {
   src:
-    "https://gforge.inria.fr/frs/download.php/33440/heptagon-1.00.06.tar.gz"
+    "https://github.com/ocaml/opam-source-archives/raw/main/heptagon-1.00.06.tar.gz"
```

Comment:
This particular one does not seem to exist

Reply:
I reverted the change, thanks!